src/Result.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+
+namespace PivotLibre\Tideman;
+
+/**
+ * Class Result
+ * Instances of this class describe the final result of the election (@see Result::getRanking()) and a few other
+ * intermediate results for reporting and visualization purposes.
+ *
+ * @package PivotLibre\Tideman
+ */
+class Result
+{
+    protected $ranking;
+    protected $winningVotesTally;
```

Comment:
Rather that having one member for winning votes and one for margins, could we have one "Pair" or "Edge" data type, and have winning and margin as members of that class?

Reply:
Yes, someday, probably not before the deadline.